tests/dummy/app/templates/application.hbs

```diff
@@ -6,4 +6,9 @@
   </div>
 {{/code-block}}
 
+<h3>With `line-numbers` plugin:</h3>
+{{#code-block class="line-numbers"}}&lt;div&gt;
```

Comment:
Or I could support both for now, by having an attribute for `plugins`, and `language`, but it basically just combines them into a `class` anyways.
But that seems unnecessary. All that would do would prevent upgrade errors for people currently using this, but it _is not_ 1.0, and not many people are using yet...

I don't know.
This is good though. Thanks! :+1: 
Can you rebase this on master?